.pipelines/js-ci.yml

```diff
@@ -41,28 +40,34 @@ parameters:
   - unit-tests
 
 steps:
-- task: NodeTool@0
-  name: NodeTool1
-  displayName: Use Node 14.x
+- task: UseNode@1
+  displayName: Use Node 20.x
   inputs:
-    versionSpec: 14.x
+    version: '20.x'
 
-- bash: |
-   npm install -g npm@8.11.0
```

Comment:
we don't need npm install step anymore?

Reply:
we were doing some non-standard things held over from when `npm` wasn't directly included in the `nodejs` release packages... but it's included now, so we don't need to manually install it.